requirements/run.txt

```diff
@@ -3,3 +3,4 @@ ruamel.yaml
 docutils
 jinja2
 cerberus
+xonsh
```

Comment:
Looks like you are importing from rever, but don't have it as a dependency

Reply:
Ah yes that could be a problem!